src/vehicle.cpp

```diff
@@ -875,6 +875,13 @@ void vehicle::use_controls()
         menu.addentry( release_remote_control, true, 'l', _("Stop controlling") );
     }
 
+    bool has_electronic_controls = false;
+
+    for( size_t p = 0; p < parts.size(); p++ ) {
```

Comment:
It's [WIP] due to this line: I don't actually want to search such flag on any part of the vehicle, instead of it, I want to check only current part with "CONTROL" flag but I don't see an elegant way to check it (and I'm sure it exists).